src/main/java/io/xcherry/core/exception/persistence/LocalAttributeNotFoundException.java

```diff
@@ -0,0 +1,8 @@
+package io.xcherry.core.exception.persistence;
+
+public class LocalAttributeNotFoundException extends RuntimeException {
```

Comment:
Do we really need this exception? I don't remember a use case

Reply:
If it's thrown when the attribute doesn't have a value, then I would prefer returning null. Because it's very common to start with empty/null and handling this exception is sometimes cumbersome

Reply:
Sounds good. For local attributes, null should be enough. While for app database, we throw not found exception if a table or column does not exist.